python/hsfs/core/external_feature_group_engine.py

```diff
@@ -134,3 +139,14 @@ def update_deprecated(self, feature_group, deprecate):
         self._feature_group_api.update_metadata(
             feature_group, copy_feature_group, "deprecate", deprecate
         )
+
+    def _get_feature_group_url(self, feature_group):
```

Comment:
Minor - Feels like we can put this method in the util module instead of having duplicates here and in the `feature_group_engine.py`

Reply:
Will refactor it